README.md

```diff
@@ -117,7 +117,7 @@ Main component for rendering the 3D PCB viewer.
 
 Props:
 
-- `soup`: (optional) An array of AnySoupElement objects representing the PCB layout.
+- `soup`: (optional) An array of AnyCircuitElement objects representing the PCB layout.
```

Comment:
We don't want to use the word `soup` any more, replace that with `circuit-json` in Readme. I think we are good to merge after that